### ISSUE TITLE
Add support for hpstr

### ISSUE THREAD

**Issue by JeremyMcCormick** [JeffersonLab/hps-mc#145]
Requested by @cbravo135 

To run:

```
hpstr hpstr/processors/config/recoTuple_cfg.py -i <input.slcio> -o <output.root>
```

**Comment by cbravo135** [comment]
In general for hpstr it will always have a format of
```hpstr [application arguments] {configuration_script.py} [arguments to configuration script]```

The arguments for the configuration script will dependent on the config, and shouldn't be assumed they will always have the same format.

**Comment by JeremyMcCormick** [comment]
Resolved by PR #170.
